library/src/main/java/com/novoda/downloadmanager/DownloadNotificationChannelCreator.java

```diff
@@ -0,0 +1,36 @@
+package com.novoda.downloadmanager;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.content.Context;
+import android.os.Build;
+
+class DownloadNotificationChannelCreator implements NotificationChannelCreator {
+
+    private final NotificationManager notificationManager;
+
+    DownloadNotificationChannelCreator(NotificationManager notificationManager) {
+        this.notificationManager = notificationManager;
+    }
+
+    private static String CHANNEL_ID;
```

Comment:
Using a `static` to ensure that the `NotificationManager.createNotificationChannel` is called once per instance of the application, rather than every time that a notification is created.